.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 language: c
 env:
     matrix:
-        - OS_TYPE=fedora
+        - OS_TYPE=fedora:28
```

Comment:
The intention of  `OS_TYPE=fedora` was to use latest fedora stable release for testing without
changing this file every time Fedora tag a release.
Why do we need to stick on Fedora 28?

Reply:
F29 Only has python3 packages.  When we specify `--with-python3` for fedora we test a combination of py2 and py3 testing which fails as py2 packages aren't available.  I'll take that out of the test and change this back.